src/constants/floors.data.ts

```diff
@@ -37,7 +427,48 @@ export const buildingFloors: BuildingFloor[] = [
       [45.457962, -73.639775]
     ],
     image: require("../../assets/floors/CC1.png"),
-    travelNodes: []
+    travelNodes: [
```

Comment:
For CC1, is it better to have 2 nodes only since it's a straight line? It might be better for our algorithm.

Also, I'm not sure if it will make any difference for the algorithm, but from the map, I feel like node 0 should be a bit more on the left. 
![image](https://user-images.githubusercontent.com/36641869/77558217-12074100-6e91-11ea-92ad-8bf8bcb0f7dd.png)

Reply:
I think we need more nodes so that the nodes can see the POI's easier so a straight polyline can be drawn. I'm open to changing it to 2 points if its easier for the algorithm.

As for the second concern, these diagrams are not exactly up to scale, when we took the points using the Picker-tool, they were more precisely taken

Reply:
Alright, I think keeping it like that is fine too :)